dispatcher/frontend/build_prod.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
 rm -r ./../backend/static/angular/
+# replace entities placeholder with backend values
+python ./gen_entities-ts.py
 ng build --prod
-mv ./dist ./../backend/static/angular
\ No newline at end of file
+mv ./dist ./../backend/static/angular
+# offer to checkout entities placeholder
+read -t 10 -p "Do you want to run git checkout on the entities placeholder? [Y/N]" answer
+if [ "$answer" != "${answer#[Yy]}" ] ;then
```

Comment:
I feel like we might simply create `categories.ts` and keep it up to date. I might be totally wrong,  but I don't think we will need to update the categories very often. Or we could fetch it from the API as well. 

Also, if we generate the file when build, it is gonna cause a bit trouble at develop time.

Reply:
Yes, this call in `build_prod.sh` was totally unnecessary. removed it.